helm/hpcc/values.yaml

```diff
@@ -513,6 +513,12 @@ sasha:
     #switchMinTime: 1
     #queues: "*"
 
+  debugplane-housekeeping: {} # NB: no properties defined, use defaults
+    #disabled: true
+    #interval: 1
```

Comment:
let's make this 24 - in keeping with the default in the code (and in other example/test yamls)